run_stf.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+# mode values: CI or AGG (for aggregating all test results after execution)
+mode="$1"
+
+export CONFIG_DIR=travis/config \
+       SCRIPT_DIR=travis/script/stf \
+       ws_dir=$PWD
+
+user="$(id -u):$(id -g)"
+source "$SCRIPT_DIR/shared_vars.sh"
+source "$SCRIPT_DIR/config_utils.sh"
+
+function restore_workspaces_permissions {
+    local user="$1"
+
+    # variables provided by shared_vars.sh
+    docker exec -t "$stf_master_name" bash -c "chown -R $user $CONTAINER_SRC $STF_CONTAINER_WS"
```

Comment:
So if I understand your PR message correctly, this script should also be usable to run the STF locally. This leads me to the following question: When run locally, does this call influence the filesystem of the user?

If this is working in the local user filesystem, doesn't it look for the dir `/home/ci/saros_src` (defined through `$CONTAINER_SRC` and the import in l.10)?
If so, this would adjust the user affiliation of a file defined through an absolute paths, meaning the file might not exist or might not be related to the process. Using absolute paths here seems dangerous to me. (Or am I reading the logic wrong?)

Have you had a look at the other solution approach setting a user id the docker root id is mapped to in the local filesystem, as suggested by @Drakulix?

Reply:
> If this is working in the local user filesystem, doesn't it look for the dir /home/ci/saros_src (defined through $CONTAINER_SRC and the import in l.10)?

The command is executed in the context of the container not in the context of your local filesystem (your filesystem is mounted into the container). 

> Have you had a look at the other solution approach setting a user id the docker root id is mapped to in the local filesystem, as suggested by @Drakulix?

I thought about the approach, but I think this would require to change our images because we had to execute the stf instances with another uid (I think this is complex, because the images are currently using a basic vnc server setup).